api/src/util/auth.rs

```diff
@@ -68,7 +67,6 @@ pub async fn oidc_request_token(
         Some(id_token) => {
             // TODO better nonce handling here
             match id_token.claims(&token_verifier, &Nonce::new("4".into())) {
-                // TODO remove clone
```

Comment:
I went through all our TODOs and removed a few that we're definitely never gonna fix.